data/remnant/remnant missions.txt

```diff
@@ -1061,138 +1061,689 @@ mission "Remnant: Learn Sign Follow Up"
+			`	Ivan leans over to his desk. "This will be for your eyes only."`
+				goto next
+			
+			label again 
+			`	"Again?" Ivan asks. He leans over to his desk. "I like that you're so interested in my research. I suppose it only makes sense with you helping me, but typically merchant captains aren't this interested in following up on odd jobs they do."`
+			
+			label next
+			apply
+				set "remnant: got deep wormhole and sprite research"
+			`	He presses a button on the desk. "Abigail, could you please make a copy of the encrypted folders 'TAB-RP,' 'WLT-RP', and 'GGA-RP' on the Institute network under my name and bring them to my office in a data chip? Thank you."`
+				goto reencrypt
+			
+			label remnant
+			apply
+				"ivan's suspicion" = ( "retrieved information" / 2 * "deep: knew of stones before scientists" ) + "deep: knew of stones before scientists" + 2 * "Deep: Remnant: Engines: declined" + 2 * "Deep: Remnant: Engines: failed" + "Deep: Remnant Surveillance: declined" + "Deep: Remnant Surveillance: failed"
+			
+			branch technology
+				has "remnant: told that deep has technology"
+			
+			branch surveillance
+				has "remnant: told about surveillance"
+			
+			choice
+				`	"I was wondering what your research on those Remnant ship scans has brought up."`
+				`	"Could I get a copy of your research on those Remnant ship scans?"`
+					goto suspicion
+			branch cubes
+				has "Deep: Remnant Surveillance: done"
+			`	"Ah. Well, there's quite a bit to talk about." Ivan reclines his chair.`
+			`	"The ship scans look like nothing we've ever seen before. The ships seem to have a sort of... organic property to them. Here, let me show you." Ivan pulls up a schematic on his computer of the Remnant ships that you scanned. You're somewhat surprised that he could pull so much information from an outfit scan. "The ships look relatively normal for the most part, but the hull of these ships have an almost organic structure to them." He zooms in on a section of the schematic that highlights the hull structure. "Now when it comes to ships designed by say Betelgeuse or even the Navy, the hull supports are arranged in triangular or square sections. But these ships have a pseudo-hexagonal design to them, completely non-uniform. It almost looks like the cell structure of some plants.`
+			`	"Aside from the strange hull structure, these ships also seem to have a considerably amount of equipment integrated directly into the ships themselves. In-built scanners and ramscoops, for example. Stuff that didn't come up on the surface of the outfit scan but that we can tell is there. The smaller of the two ships you scanned also has some sort of in-built device that we can't make any sense of. It doesn't look like any sort of device that we've seen before, but it takes up a considerable portion of the ship's structure, so it must have some purpose. We'd need to get a closer look to figure out what it is."`
```

Comment:
```suggestion
			`	"Aside from the strange hull structure, they also seem to have a considerable amount of equipment integrated directly into the ships themselves. Built-in scanners and ramscoops, for example. Stuff that didn't come up on the surface of the outfit scan but that we can tell is there. The smaller of the two ships you scanned also has some sort of built-in device that we can't make any sense of. It doesn't look like any sort of device that we've seen before, but it takes up a considerable portion of the ship's structure, so it must have some purpose. We'd need to get a closer look to figure out what it is."`
```
Did a little googling since I rarely hear `in-built`, and it seems to be a primarily British synonym to "built-in", so I'd suggest using "built-in" instead. Besides that, just a little fix and removal of another double `the(se) ships`.